### ISSUE TITLE
關於 readme 結果的一些問題

### ISSUE THREAD

**Issue by b06b01073** [ricky-696/AICUP_Baseline_BoT-SORT#3]
您好
想要請教一下在 readme 中 **Visualization results on AICUP train_set** 的影片以及 **Evaluate your submission** 的[表格](https://github.com/ricky-696/AICUP_Baseline_BoT-SORT/blob/main/demo_readme/eval_res.png)是怎麼得到的呢?

比如說跑出這些結果的 model 有沒有進行過 fine-tuning 等等? 謝謝。

**Comment by ricky-696** [comment]
哈囉~

此表格的結果是使用`yolov7-e6e.pt`作為物件偵測的權重，並使用我們Fine-tune過的ReID模型提取外觀特徵得到的，ReID模型並沒有調整任何的超參數，照著[Readme](https://github.com/ricky-696/AICUP_Baseline_BoT-SORT?tab=readme-ov-file#train-the-reid-module-for-aicup)的步驟訓練應該就可以復現此結果。

**Comment by b06b01073** [comment]
請問影片所呈現的結果也是嗎?

**Comment by b06b01073** [comment]
另外想要請教一下，在 ground truth 當中有些停在路邊的車輛並沒有被標註出來，請問相關的 labeling 標準是甚麼呢? 實際上是否會出現像是官方給的 training set 與實際上競賽的 public 或 private dataset 對於路邊車輛的檢測標準不一，導致資料分布的情形出現很大的歧異呢? 謝謝。

**Comment by ricky-696** [comment]
1. 影片呈現的結果是有用過比賽資料集去Fine-tune過後的結果，印象中Fine-tune YOLO過後分數比沒Fine-tune差。
2. Labeling的標準一言難盡，本身資料集就有非常多的Noisy Label，我建議你可以參考一些專門處理Noisy Label的論文，最終的testing set會盡量把停在路邊的車輛標記出來。

**Comment by b06b01073** [comment]
了解，感謝回覆

**Comment by ricky-696** [comment]
@b06b01073 抱歉兄弟，更正一下，經過測試有用我們資料集Fine-tune過後的YOLO，效果會比只使用Coco Pretrain YOLO的效果還要好。

**Comment by b06b01073** [comment]
了解，感謝通知
另外，想要請問一下在 camera 7 中被藍色圈起來區塊的車輛是否需要被考慮
![7_00289](https://github.com/ricky-696/AICUP_Baseline_BoT-SORT/assets/56951221/969c6163-61cf-43e2-b82c-56c2c76a93db)

**Comment by ricky-696** [comment]
@b06b01073 我們在Test set有對紅色區塊做標記，所以需要考慮那部分~
![image](https://github.com/ricky-696/AICUP_Baseline_BoT-SORT/assets/62836757/d7dfa984-ab42-47e0-845b-a2c8bd2794c6)
